src/ripple/ledger/View.h

```diff
@@ -341,6 +341,12 @@ bool amendmentRIPD1141 (NetClock::time_point const closeTime);
 NetClock::time_point const& amendmentRIPD1141SoTime ();
 bool amendmentRIPD1141 (NetClock::time_point const closeTime);
```

Comment:
Consider deleting these two lines since the redundant declarations are unnecessary.